amida-tech/indaba-client
# [INBA-984] Fix allowing the user to choose the same start and end date

#### What does this PR do?
Fix for the issue @Arghast found, allowing a user to choose the same start and end date for a stage.

#### Related JIRA tickets:
INBA-984

#### How should this be manually tested?
Go into Create Project Wizard. Go through the steps to add stages. Try keeping title blank and saving. You should be stopped with a red field for title, instructing you to fill it out. Then go to the start date. Tryp Ctrl + A to select it all and delete it. Attempt to submit it, and you should get a red title for the date.

Add a title and try to save, confirm you still have the red title for date. Add a date and remove the title, confirm you still have to enter the title. Try again removing end date, and the user should be forced to the start date. Fill out all info and save the stage. Click on it again and confirm all details are correct. Go to Projects, reload the page. Click on the stage and confirm its correct. 

Repeat for Project Management and be sure to do both a new stage and editing an existing one. Finally, try deleting stages to confirm the usual confirmations appear. 

#### Background/Context

#### Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4026454/52038134-41c1ef80-24ff-11e9-90f2-7c9a1becf78a.png)
